main.ts

```diff
@@ -325,6 +330,19 @@ class AutoArchiveSettingTab extends PluginSettingTab {
 				})
 				.setClass("aa-folder-search");
 
+			new Setting(containerEl)
```

Comment:
Either make the setting description sound like it only excludes one tag (like it currently does), or make this feature support multiple tags. Also, please call them "tags" instead of "labels" for consistency.